Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -31,10 +31,53 @@
 // Significantly modified for re-motion
 // ************************************************
 
+interface JQueryStatic { Autocompleter: any; }
```

Comment:
I don't think I fully understand. Would you like me to change every invocation of `$.Autocompleter` so that it's not a static member of JQueryStatic  anymore? Would you like me to change the name `Autocompleter` to something else?

Reply:
No, it's a naming thing. You added a globally accessible type that is specific to BocAutoCompleteReferenceValue but uses a generic name. Therefor, someone else could wish to introduce a type of the same name and run into problems.

Reply:
But it needs to be named `JQueryStatic` so we can access `Autocompleter` on `$`. Perhaps I'm still misunderstanding you

Reply:
Ah, okay. @a-ctor I'd like some more details on this.

@Stefan-Ilic so, maybe it's just the 'regular' types, then, like `Options` in line 36 that need renaming.